### PR TITLE
i18n(fr): Add `db.mdx` `studio.mdx` `cli-reference.mdx` (Add )Astro Studio) + Update `nav.ts` and `ui.ts`

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
Add `db.mdx` `studio.mdx` `cli-reference.mdx` (Add )Astro Studio) + Update `nav.ts` and `ui.ts`

For the `astro-db.mdx` file, see https://github.com/withastro/docs/pull/7527

#### Related issues & labels (optional)

- Suggested label: i18n

🚀🥳

### CODE REVIEW

**Comment by thomasbnt** [comment]
Drafted,

Miss `guides/astro-db.mdx` file. I'm translating now.

**Comment by thomasbnt** [comment]
> Thanks @thomasbnt, could you confirm this is all updated and update those pages if necessary? I believe there are a few missing changes, then we should be good to merge it.

All right! Everything has been updated!

File: src/content/docs/fr/guides/integrations-guide/db.mdx [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+
+Vérifier s'il y a des différences entre la configuration de la base de données locale et celle de la base de données distante. Ceci est automatiquement lancé par `astro db push`. `verify` comparera votre fichier local `db/config.ts` avec la base de données distante et vous avertira si des changements sont détectés.
+
+### `astro db execute <file-path>`
+
+**Flags :**
+
+- `--remote` Exécuter sur la base de données de votre projet Studio. Ignorer pour l'exécuter sur votre serveur de développement.
+
+Exécute un fichier `.ts` ou `.js` pour lire ou écrire dans votre base de données. Ceci accepte un chemin de fichier comme argument, et supporte l'utilisation du module `astro:db` pour écrire des requêtes sûres. Utilisez l'option `--remote` pour lancer l'exécution sur la base de données de votre projet Studio, ou ignorez l'option pour lancer l'exécution sur votre serveur de développement. Voir comment utiliser les [données sur le développement des seeds](/fr/guides/astro-db/#seed-your-database) pour un exemple de fichier.
```
Comment (pioupia) [inline src/content/docs/fr/guides/integrations-guide/db.mdx:217]:
```suggestion
Exécute un fichier `.ts` ou `.js` pour lire ou écrire dans votre base de données. Cette commande accepte un chemin de fichier comme argument, et supporte l'utilisation du module `astro:db` pour écrire des requêtes sûres. Utilisez l'option `--remote` pour lancer l'exécution sur la base de données de votre projet Studio, ou ignorez l'option pour lancer l'exécution sur votre serveur de développement. Voir comment utiliser les [données sur le développement des seeds](/fr/guides/astro-db/#seed-your-database) pour un exemple de fichier.
```

File: src/content/docs/fr/reference/cli-reference.mdx [hunk 2]
```diff
@@ -531,3 +532,17 @@ const exitCode = await sync({
 
 process.exit(exitCode)
 ```
+
+## Astro Studio CLI
+
+### `astro login`
+
+S'authentifier avec Astro Studio. Ceci est nécessaire pour exécuter toutes les commandes de gestion de base de données, y compris [`astro link`](#astro-link).
```
Comment (pioupia) [inline src/content/docs/fr/reference/cli-reference.mdx:540]:
```suggestion
S'authentifier avec Astro Studio. Cette action est nécessaire pour exécuter toutes les commandes de gestion de base de données, y compris [`astro link`](#astro-link).
```

**Review by pioupia — APPROVED** [review]
LGTM!

**Comment by thomasbnt** [comment]
Merci @pioupia pour la review!

**Review by pioupia — APPROVED** [review]

**Comment by thomasbnt** [comment]
> 
> 
> 
> ## Lunaria Status Overview
> 
> 
> 🌕 **This pull request will trigger status changes.**
> 
> <details>
> <summary>Learn more</summary>
> 
> By default, every PR changing files present in the [Lunaria configuration's `files` property](https://lunaria.dev/reference/configuration/#files-required) will be considered and trigger status changes accordingly.
> 
> You can change this by adding one of the keywords present in the [`ignoreKeywords` property in your Lunaria configuration file](https://lunaria.dev/reference/configuration/#ignorekeywords) in the PR's title (ignoring all files) or by [including a tracker directive](https://lunaria.dev/guides/tracking/#tracker-directives) in the merged commit's description. 
> </details>
> 
> ### Tracked Files
> 
> 
> | File                                                                                                                                                                                                     | Note                                                  | Locale |
> | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------- | ------ |
> | [src/content/docs/fr/guides/integrations-guide/db.mdx](https://github.com/withastro/docs/blob/1800871d2d1456f6802df80842440c31057d15af/src%2Fcontent%2Fdocs%2Ffr%2Fguides%2Fintegrations-guide%2Fdb.mdx) | Localization added, will be marked as complete. 🔄️   | fr     |
> | [src/content/docs/fr/recipes/studio.mdx](https://github.com/withastro/docs/blob/1800871d2d1456f6802df80842440c31057d15af/src%2Fcontent%2Fdocs%2Ffr%2Frecipes%2Fstudio.mdx)                               | Localization added, will be marked as complete. 🔄️   | fr     |
> | [src/content/docs/fr/reference/cli-reference.mdx](https://github.com/withastro/docs/blob/1800871d2d1456f6802df80842440c31057d15af/src%2Fcontent%2Fdocs%2Ffr%2Freference%2Fcli-reference.mdx)             | Localization changed, will be marked as complete. 🔄️ | fr     |
> | [src/i18n/fr/nav.ts](https://github.com/withastro/docs/blob/1800871d2d1456f6802df80842440c31057d15af/src%2Fi18n%2Ffr%2Fnav.ts)                                                                           | Localization changed, will be marked as complete. 🔄️ | fr     |
> | [src/i18n/fr/ui.ts](https://github.com/withastro/docs/blob/1800871d2d1456f6802df80842440c31057d15af/src%2Fi18n%2Ffr%2Fui.ts)                                                                             | Localization changed, will be marked as complete.     | fr     |
> 
> <details>
> <summary>Warnings reference</summary>
> 
> | Icon | Description                                                                                                                                       |
> | ---- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
> | 🔄️  | The source for this localization has been updated since the creation of this pull request, make sure all changes in the source have been applied. |
> 
> </details>



Mais c'est quoi ça? 😳

File: src/content/docs/fr/guides/integrations-guide/db.mdx
[the remark this answers is not in the document]
Comment (thomasbnt) [reply src/content/docs/fr/guides/integrations-guide/db.mdx:1]: Thanks, I will update that today

[the remark this answers is not in the document]
Comment (thomasbnt) [reply src/content/docs/fr/guides/integrations-guide/db.mdx:1]: Pushed!

File: src/content/docs/fr/recipes/studio.mdx
Comment (yanthomasdev) [inline src/content/docs/fr/recipes/studio.mdx:1]: Didn't notice the first time, sorry, but we are also missing this commit here: https://github.com/withastro/docs/commit/bc1e565fc842dd5ccd9a0f2d9c43813dfce8da21#diff-da87636ba512eac501cf82f2e6affa759839803533a14dddc4c80aaec2e55bc7

**Review by yanthomasdev — APPROVED** [review]
LGTM! Good work 🙌

**Pull request merged**